packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -252,6 +264,15 @@ export interface DockerBuildOptions {
    * @default - no build args
    */
   readonly buildArgs?: { [key: string]: string };
+
+  /**
+   * The image name to include in the asset hash. This option can be used to
+   * stabilize the assetHashes emitted by bundlers that are built locally.
+   *
+   * @default - The image name used by the bundler will be the same as the image
+   * name.
+   */
+  readonly assetHashName?: string;
```

Comment:
Where is this used?

Reply:
I've removed it. It was an unintended leftover from an earlier commit in this PR. Sorry about that.